browser/src/control/Control.JSDialog.js

```diff
@@ -384,8 +384,9 @@ L.Control.JSDialog = L.Control.extend({
 
 		this.addFocusHandler(instance); // Loop focus for all dialogues.
 
+		const removeChar = new RegExp("[()]", 'g');
 		var clickToCloseId = instance.clickToClose
-			? instance.clickToClose.replaceAll(' ', '') : null;
+			? instance.clickToClose.replaceAll(' ', '').replaceAll(removeChar, match => '\\' + match) : null;
```

Comment:
- can we put that in a method for readability? eg. `sanitizeElementId`
- `removeChar` is misleading name now
- also would be good to put human-readable-comment next to every step: what the regexp does (as not all people have interpreter in the eyes :) )

like:
```
/// function to prepare id for usage in the querySelector looking for an element id
sanitizeElementId(id: string): string {
    const toEscapeRegExp = new RegExp('[()]', 'g'); // match parenthesis

    return id
            // remove spaces
            .replaceAll(' ' , '')
            // escape forbidden characters
            .replaceAll(toEscapeRegExp, match => '\\' + match);
}
```